CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Changed Invalidation Jobs throughout (TO, TP, T3C, etc.) to account for the ability to do both REFRESH and REFETCH requests for resources.
 - The `admin` Role is now always guaranteed to exist, and can't be deleted or modified.
 - Updated `t3c-apply` to reduce mutable state in `TrafficOpsReq` struct.
+- Traffic Portal: Changed the warning text for creating a Content Invalidation job to add clarity and to emphasize severity. 
```

Comment:
This hasn't actually changed though, has it? I see the line 
> - Changed Invalidation Jobs throughout (TO, TP, T3C, etc.) to account for the ability to do both REFRESH and REFETCH requests for resources.

above, so that hasn't actually been released, thus the warning text is not now different in a new release than it has been before. You know what I'm trying to say? I feel like these are arcane cascading clauses and I'm getting lost in them.

Reply:
Good call. Removed.